### ISSUE TITLE
Accessibility features not available

### ISSUE THREAD

**Issue by Srihan-Yeleswarapu** [Srihan-Yeleswarapu/Computer-Game-and-Simulation-Programming#2]
We need to add a High Contrast mode and ensure all the text is distinct from the backgrounds. We also need to make sure clicking is available where it should be allowed and keyboard should be availbale where it should be allowed. This makes an accessibility benefit. 

UX Design - User journey, design rationale, accessibility features 

5 points: UX design presented, withdetails about the design rationale, user journey, and **accessibility** features highlighted

0 points: No UX design presented
